src/core/packages/saved-objects/base-server-internal/src/mappings/types.ts

```diff
@@ -60,7 +61,15 @@ export interface IndexMapping {
 export type IndexTypesMap = Record<string, string[]>;
 
 /** @internal */
-export interface V2AlgoIndexMappingMeta {
+export interface CommonAlgoIndexMappingMeta {
+  /**
+   * The current virtual version of the mapping of the index.
+   */
+  mappingVersions?: VirtualVersionMap;
+}
```

Comment:
Just extracting `mappingVersions` in a common interface, as it is the same for V2 and ZDT.